### PR TITLE
Replace `hpxml_inputs.tsv` with `hpxml_inputs.json`

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Using a JSON makes it easier to see changes and deal with conflicts. Also, the JSON re-introduces the concept of creating a test file from a parent test file, which A) minimizes the amount of properties needed to describe a given test file and B) helps ensure test files stay in sync.

All test/sample files are 100% unchanged.

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] Schematron validator (`EPvalidator.xml`) has been updated
- [ ] Sample files have been added/updated (via `tasks.rb`)
- [ ] Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests` and/or `workflow/tests/hpxml_translator_test.rb`)
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: tasks.rb [hunk 1]
```diff
@@ -9,37 +9,43 @@
...
+    json_input = { 'hpxml_path' => hpxml_path }
+    for parent_json_filename in parent_json_filenames.reverse
+      json_input.merge!(json_inputs[parent_json_filename])
     end
+    json_input.merge!(json_inputs[hpxml_filename])
+    json_input.delete('parent_json')
 
     measures = {}
-    measures['BuildResidentialHPXML'] = [csv_row]
-    # measures['BuildResidentialScheduleFile'] = [sch_args] if !sch_args.empty?
```
Comment (joseph-robertson) [inline tasks.rb:42]: We no longer call `BuildResidentialScheduleFile` to re-generate csv files in `HPXMLtoOpenStudio/resources/schedule_files/`? Do we need to ensure these files are kept up-to-date?

Comment (shorowit) [reply tasks.rb:42]: Good question. I think I lost that capability by accident. Let me see if I can somewhat easily bring it back.

Comment (shorowit) [reply tasks.rb:42]: Addressed in last commit.

File: tasks.rb [hunk 2]
```diff
@@ -47,6 +49,16 @@ def create_hpxmls
...
+    # Re-generate stochastic schedule CSV?
+    csv_path = json_input['schedules_filepaths'].to_s.split(',').map(&:strip).find { |fp| fp.include? 'occupancy-stochastic' }
+    if (not csv_path.nil?) && (not schedules_regenerated.include? csv_path)
+      sch_args = { 'hpxml_path' => hpxml_path,
+                   'output_csv_path' => csv_path,
+                   'hpxml_output_path' => hpxml_path }
+      measures['BuildResidentialScheduleFile'] = [sch_args]
+      schedules_regenerated << csv_path
+    end
+
```
Comment (joseph-robertson) [inline tasks.rb:61]: 👍

**Review by joseph-robertson — APPROVED** [review]

**Pull request merged**